### PR TITLE
WIP [DO NOT MERGE] Adding base logic for Proof of Concept between Polling Service & Magento CH-28362

### PR DESCRIPTION
**Pull request by CTDoriott**

### CODE REVIEW

File: Cron/Test.php [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+                return;
+            }
+
+            $fileList = array_map('basename', $fileList);
+            $fileList = array_map('intval', $fileList);
+            sort($fileList);
+            for ($i=0; $i <count($fileList)-1; $i++) {
+                $file = self::SOURCE_DIRECTORY.$fileList[$i];
+                $this->handleFile($file);
+            }
```
Comment (davehensleyns8) [inline Cron/Test.php:64]: Are you intentionally skipping the last file in the list? If not, it's probably cleaner to use a `foreach` here:
```
foreach ($fileList as $file) {
    $this->handleFile(self::SOURCE_DIRECTORY . $file);
}
```

File: Cron/Test.php [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+    /**
+     * Logging attribute
+     * @var LoggerInterface $logger
+     */
+    protected $logger;
+
+    /**
+     * Default constructor
+     *
+     * @param Psr\Log\LoggerInterfac $logger
```
Comment (davehensleyns8) [inline Cron/Test.php:29]: `@param LoggerInterface $logger The logger`

File: Cron/Test.php [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+     */
+    protected function handleFile(string $file) : bool
+    {
+        $logData = file($file);
+        foreach ($logData as $line) {
+            $this->logger->debug($line);
+        }
+        unlink($file);
+
+        return true;
```
Comment (davehensleyns8) [inline Cron/Test.php:85]: Suggestion: Maybe should have this return `false` if the file doesn't exist or isn't readable, etc.?

**Comment by crfroehlich** [comment]
Looks good to me.

**Comment by rob-ns8** [comment]
Looks good to me too.

**Pull request closed without merging**